### DISCUSSION TITLE
documentation for create plugin ?

### DISCUSSION THREAD

**Discussion by sadoj-kuro** [discord-tickets/bot#351] [Support (Q&A)] [answered]
I can't find any documentation regarding how to create a plugin. I would like to make the problem of it being that I do not know if it is enough to create the file in the plugin folder or if it is necessary to go and declare things somewhere

**Comment by eartharoid** [comment, marked as answer]
The plugins feature is incomplete and barely functional. I won't be writing any documentation for it it because 

1. There's not much to write about, as in its current state there's very little you can do with the plugin system 
2. v4 will be released soon, and plugins are being dropped. Transcripts will be built-in

**Reply by sadoj-kuro** [reply]
thank im waiting to V4
